templates/agents.html

```diff
@@ -308,12 +310,22 @@ <h3 id="modal-paw"></h3>
         return [];
     }
 
+    function createContactDropdown(data){
+        let str = `<option selected=\"selected\" value=${data['contact']}>${data['contact']}</option>\n`;
+        data['c2'].forEach(function(contact){
+            if(contact === data['contact']) return;
+            str += `<option>${contact}</option>\n`;
+        });
+        return str;
+    }
+
     function showAgentInfo(paw){
         function agentInfoCallback(data){
             let parent = $('#agent-modal');
             let agent = data[0];
+            let dropdown = createContactDropdown(agent);
             parent.find('#modal-paw').html(agent['paw']);
-            parent.find('#modal-contact').html(agent['contact']);
+            parent.find('#modal-contact').html(dropdown);
```

Comment:
this works-- but doing it in raw HTML can be a bit hacky. html is good if we're handling text or something like displaying a Unicode character in an HTML DOM object. for more complex JS objects (like drop-downs) we should leverage jQuery. 

example:
```
$('#modal-contact').append(`<option value="${optionValue}">some contact here</option>`);
```
You can loop through all the contacts returned in the data[0] object and append them on the select

let's use the jQuery library that we have imported to do this a bit cleaner.